packages/azure-services/src/azure-cosmos/cosmos-client-wrapper.ts

```diff
@@ -4,7 +4,7 @@ import * as cosmos from '@azure/cosmos';
 import { System } from 'common';
 import { inject, injectable } from 'inversify';
 import { isEmpty } from 'lodash';
-import { Logger } from 'logger';
+import { ContextAwareLogger, Logger } from 'logger';
```

Comment:
remove?

Reply:
i have removed it. But, we need to enable nounusedLocals tsconfig option to catch all issues & fix them. i will do that in another PR.